### PR TITLE
fix(manager): add links to .list/.repo files for release 3.3

### PR DESCRIPTION
**Pull request by mikliapko**
Since the Manager 3.3 has been released, the manager_versions.yaml is updated to have the links to .repo and .list files for this release.

Together with that, the links for old release 3.0 have been removed.

### Testing

- [ ]

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Review by karol-kokoszka — APPROVED** [review]

File: defaults/manager_versions.yaml [hunk 1]
```diff
@@ -7,6 +7,14 @@ manager_repos_by_version:
     ubuntu20: 'https://downloads.scylladb.com/manager/deb/unstable/unified-deb/master/latest/scylla-manager.list'
     ubuntu22: 'https://downloads.scylladb.com/manager/deb/unstable/unified-deb/master/latest/scylla-manager.list'
     ubuntu24: 'https://downloads.scylladb.com/manager/deb/unstable/unified-deb/master/latest/scylla-manager.list'
+  "3.3":
+    centos7: 'http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylladb-manager-3.3.repo'
+    centos8: 'http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylladb-manager-3.3.repo'
```
Comment (soyacz) [inline defaults/manager_versions.yaml:12]: no need for rocky?

**Review by soyacz — APPROVED** [review]
LGTM

File: defaults/manager_versions.yaml (hunk 1 above)
Comment (mikliapko) [reply defaults/manager_versions.yaml:12]: yes, for now rocky is mapped on centos, we're going to refactor it soon when get rid of Centos for Azure

**Review by rayakurl — APPROVED** [review]
LGTM

**Comment by roydahan** [comment]
@mikliapko what about backporting SCT to use manager-3.3 in all live branches?
5.4, 6.0, 6.1, 2024.1, 2023.1

**Pull request merged**